lib/node_modules/@stdlib/math/base/special/betainc/benchmark/c/cephes/benchmark.c

```diff
@@ -95,19 +95,22 @@ static double rand_double( void ) {
 */
 static double benchmark( void ) {
 	double elapsed;
-	double x;
-	double a;
-	double b;
+	double x[ 100 ];
+	double a[ 100 ];
+	double b[ 100 ];
```

Comment:
```suggestion
	double x[ 100 ];
	double a[ 100 ];
	double b[ 100 ];
	double elapsed;
```

Array variables should be declared before single variables.